ui/perfherder/alerts/AlertHeader.jsx

```diff
@@ -121,6 +122,11 @@ const AlertHeader = ({
           />
         </Col>
       </Row>
+      <Row>
+        <Col className="p-0" xs="auto">
```

Comment:
Could we guard this column's creation with a truthy `performance_tags`, like we did with `bugNumber` above?